cms/envs/aws.py

```diff
@@ -494,4 +494,12 @@
 AZURE_ACCOUNT_NAME = AUTH_TOKENS.get('AZURE_ACCOUNT_NAME', None)
 AZURE_ACCOUNT_KEY = AUTH_TOKENS.get('AZURE_ACCOUNT_KEY', None)
 AZURE_CONTAINER = AUTH_TOKENS.get('AZURE_CONTAINER', None)
+
+if AUTH_TOKENS.get('RG_SENTRY_DSN', None):
```

Comment:
Why do you want to user deprecated client?

Reply:
`urllib3` module used in `edx-platform` is not compatible with `sentry_sdk` module
```
>>> import sentry_sdk
>>> sentry_sdk.init("https://***@sentry.raccoongang.com/**")
Traceback (most recent call last):
  File "<console>", line 1, in <module>
  File "/edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/sentry_sdk/hub.py", line 67, in init
    client = Client(*args, **kwargs)
  File "/edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/sentry_sdk/client.py", line 76, in __init__
    self.transport = make_transport(options)
  File "/edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/sentry_sdk/transport.py", line 225, in make_transport
    return transport_cls(options)
  File "/edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/sentry_sdk/transport.py", line 80, in __init__
    self._retry = urllib3.util.Retry()
AttributeError: 'module' object has no attribute 'Retry'
```